src/NServiceBus.Core.Tests/Pipeline/Outgoing/AttachSenderRelatedInfoOnMessageTests.cs

```diff
@@ -23,7 +25,7 @@ public async Task Should_set_the_time_sent_headerAsync()
         [Test]
         public async Task Should_not_override_the_time_sent_headerAsync()
         {
-            var timeSent = DateTime.UtcNow.ToString();
+            var timeSent = DateTimeOffsetHelper.ToWireFormattedString(DateTimeOffset.UtcNow);
```

Comment:
@TravisNickels is this the change that you were talking about? I'm worried the point of this test is to ensure that if the message has some weird non-conforming header, as long as it's parseable to a DateTime it won't be modified. Changing the format of the test header from `2/14/2022 9:15:21 PM` to `2022-02-14 21:15:21:174655 Z` will not do that.

Granted, if I'm right, it would sure be nice if there were a comment to say that.

Reply:
Set timeSent back to DateTime.UtcNow.ToString();